src/components/common/forms/SignInForm.tsx

```diff
@@ -48,16 +49,14 @@ function SignInForm() {
           register={register}
           error={!isValid}
           onChange={() => clearErrors()}
-          name="email"
-        />
+          name="email" placeholder={""}        />
```

Comment:
Do you need to send an empty placeholder? 
You can indicate that it's an optional prop, and set it's default as empty in the Input component.